docs/examples/rx/03_listen_to_balance_change/index.js

```diff
@@ -8,26 +9,29 @@ async function main () {
   // Create an await for the API
   const api = await ApiRx.create().toPromise();
 
-  // Retrieve the initial balance. Since the call has no callback, we can use the toPromise()
-  // method on the observable together with the first() operator.
-  let previous = await api.query.balances.freeBalance(ALICE).pipe(first()).toPromise();
-
-  console.log(`${Alice} has a ${previous} balance`);
-  console.log(`You may leave this example running and start example 06 or transfer any value to ${Alice}`);
-
-  // Here we subscribe to any balance changes and update the on-screen value
-  api.query.balances.freeBalance(Alice).subscribe((balance) => {
-    // Calculate the delta
-    const change = current.sub(previous);
-
-    // Only display positive value changes (Since we are pulling `previous` above already,
-    // the initial balance change will also be zero)
-    if (change.isZero()) {
-      return;
-    }
-    previous = current;
-    console.log(`New balance change of: ${change}`);
-  });
+  // Here we subscribe to any balance changes and update the on-screen value.
+  // We're using RxJs pairwise() operator to get the previous and current values as an array.
+  api.query.balances.freeBalance(Alice)
+    .pipe(
+      // since pairwise only starts emitting values on the second emission, we prepend an
+      // initial value with the startWith() operator to be able to also receive the first value
+      startWith('first'),
+      pairwise()
+    )
+    .subscribe((balance) => {
+      if (balance[0] === 'first') {
+        // Now we know that if the previous value emitted as balance[0] is `first`,
+        // then balance[1] is the initial value of Alice account.
+        console.log(`<b>Alice</b> ${Alice} has a balance of ${balance[1]}`);
+        console.log('You may leave this example running and start the "Make a transfer" example or transfer any value to Alice address');
+        return;
+      }
+      const change = balance[1].sub(balance[0]);
```

Comment:
bikeshedding a bit - but generally leave a space before/after the const blocks. So rather -

```
if (balance....) {
...
}

const change = ...

/// only display...
if (!change...) {
...
}
```

(extra whitespace doesn't cost anything, but does help in quickly recognising the sections.

Reply:
Personally I'm not a big fan of too many whitespaces since they force me to scroll vertically sooner than needed, but I get the point!
I think I developed an unconscious habit of getting rid of as many of them as possible :D

I added one here.